ibdgc.datacommons.io/portal/gitops.json

```diff
@@ -74,15 +74,16 @@
       ],
       "customHomepageChartConfig": [
         {
-          "chartType": "horizontalStackedBar",
+          "chartType": "horizontalGroupedBar",
           "dataType": "participant",
           "yAxisProp": "country",
           "xAxisProp": "ibd_affection_status",
```

Comment:
according to the doc `"dataTypePlural": "cases" // optional, string, by default it will be `${xAxisProp}s``
idk where it is used but are we going to end up with `ibd_affection_statuss` since it's not specified?

Reply:
oh there's a typo in README it should be `${dataType}s`, will submit a PR to fix that.